### ISSUE TITLE
Compile error when setting camera LookAt 

### ISSUE THREAD

**Issue by IRobot1** [nartc/angular-three#116]
```html
<ngt-canvas [camera]="{ position: [0,2, 2], lookAt: [0, 0, 0], fov: 45 }">
```

Error: src/app/examples/vr/vr.component.html:4:55 - error TS2322: Type '[number, number, number]' is not assignable to type '(((vector: number | Vector3, y?: number | undefined, z?: number | undefined) => void) & number) | (((vector: number | Vector3, y?: number | undefined, z?: number | undefined) => void) & Vector3) | (((vector: number | Vector3, y?: number | undefined, z?: number | undefined) => void) & [x: ...]) | (((vector: number | ...'.
  Type '[number, number, number]' is not assignable to type '((vector: number | Vector3, y?: number | undefined, z?: number | undefined) => void) & NgtTriple'.
    Type '[number, number, number]' is not assignable to type '(vector: number | Vector3, y?: number | undefined, z?: number | undefined) => void'.
      Type '[number, number, number]' provides no match for the signature '(vector: number | Vector3, y?: number | undefined, z?: number | undefined): void'.

**Comment by nartc** [comment]
ugh, I missed this issue in latest release. I'll fix this tomorrow. Sorry for the inconvenience

**Comment by nartc** [comment]
Released in 5.2.2 with `[lookAt]` on `<ngt-canvas>` instead
